src/commands/cicdPipeline.ts

```diff
@@ -0,0 +1,68 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+import { CloudExplorerV1 } from 'vscode-kubernetes-tools-api';
+
+const deployToAzureExtensionId = 'ms-vscode-deploy-azure.azure-deploy';
+const configurePipelineCommand = 'configure-cicd-pipeline';
+const azurePortalUrl = 'https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/';
```

Comment:
🐛 I think `ms.portal.azure.com` the internal preview version of the portal.

* For external it should be `portal.azure.com`. (Please double check \ Test)